src/rust_bindings/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "chiavdf"
+version = "1.1.4"
+edition = "2021"
+license = "Apache-2.0"
+description = "Bindings to the chiavdf C++ library for vdf verification."
+authors = ["Brandon Haggstrom <b.haggstrom@chia.net>"]
+homepage = "https://github.com/Chia-Network/chiavdf"
+repository = "https://github.com/Chia-Network/chiavdf"
+links = "chiavdf"
+
+[dependencies]
+link-cplusplus = "1.0.9"
```

Comment:
the name of this suggests that it would be a dev-dependency. does it generate bindings?

Reply:
It can't be a dev dependency since it must be `extern crate` to run its build script, which only links libc++ or stdlibc++